README.md

```diff
@@ -3,6 +3,87 @@ Checkmate
 
 A service for checking URLs are safe.
 
+API
```

Comment:
Can we move the **API** and **Configuration** sections to _below_ **Installing Checkmate in a development environment**? All our README's start with the same dev install instructions at the top (after a one-line description of what the app is) and then other stuff (varies from app to app) below. How to install it is usually the main / first thing that people are looking for. And there's just a high value to consistency here when we have several apps: that they all install in the same way, and the install instructions can always be found in the same place (and in the same format)

Reply:
I think we might want to put a heading at the top and move it to a separate file. I think it's the first thing people look for _the first time_ they come to the app, but the last thing they care about thereafter.